tokio/tests/time_interval.rs

```diff
@@ -470,8 +470,25 @@ async fn stream_with_interval_poll_tick_no_waking() {
     assert_eq!(items, vec![]);
 }
 
-#[tokio::test(start_paused = true)]
+#[tokio::test]
```

Comment:
Why can't this test use `start_paused`?

Reply:
We need a bit of delay between two subsequent `poll_tick` calls (5ms) to enter the misstick behavior and trigger the overflow panic

Reply:
In general, a test that takes 2 seconds to complete is not acceptable. I'm quite certain that the test works just fine with `start_paused` - the parameter should not change the behavior of tests that rely on time other than making them run faster.